dev/system-test/firestore.ts

```diff
@@ -1820,6 +1794,29 @@ describe('Query class', () => {
           unsubscribe();
         });
     });
+
+    it('orders limitToLast() correctly', async () => {
```

Comment:
Can this be moved to other limitToLast tests?

Reply:
This test is an integration test (in system-test/) and thus cannot be moved.